website/blog/2024-06-26-llm-as-judge/index.md

```diff
@@ -0,0 +1,522 @@
+---
+title: LLM as judge
+description: Perform LLM Evaluations with custom metrics
+slug: llm-as-judge
+authors: [pedro-azevedo,rahul-pandey]
+tags: [genai,mlflow-evalaute]
+thumbnail: img/blog/llm-as-judge.png
+---
+
+In this blog post, we'll dive on a journey to revolutionize how we evaluate language models. We'll explore the power of MLflow Evaluate and harness the capabilities of Large Language Models (LLMs) as impartial judges. By the end, you'll learn how to create custom metrics, implement LLM-based evaluation, and apply these techniques to real-world scenarios. Get ready to transform your model assessment process and gain deeper insights into your AI's performance!
+
+## The Challenge of Evaluating Language Models
+
+Imagine you're part of a global travel agency, "WorldWide Wandercorp," that's expanding its reach to Spanish-speaking countries. Your team has developed an AI-powered translation system to help create culturally appropriate marketing materials and customer communications. However, as you begin to use this system, you realize that traditional evaluation metrics fall short in capturing the nuances of language translation, especially when it comes to preserving cultural context and idiomatic expressions. You want to understand the materials in which your translation is failing so you can improve the model or manually translate them.
+
+This is where MLflow Evaluate and LLMs as judges come into play. Let's dive into how these tools can help solve Worldwide WanderAgency's challenges and improve their AI-driven communication strategy.
```

Comment:
-Try to be clear about when you're talking about a specific function like [mlflow.evaluate()](https://mlflow.org/docs/latest/python_api/mlflow.html#mlflow.evaluate) vs. [MLflow llm evaluation more generally](https://mlflow.org/docs/latest/llms/llm-evaluate/index.html), which encompasses multiple functions/tools/metrics.

Reply:
A good way to frame this is to force yourself to provide a link to an API. [mlflow.evaluate](<link to the API docs page>) so that you're elucidating a usage of an API or a general module-level concept [MLflow Evaluate](<link to the landing page in the MLflow docs that covers GenAI evaluation>). This can help to frame your references so that readers can go and learn more if they choose :)

Reply:
I've added the urls and tried to make this a bit more clear, let me know if it's still unclear and I will re-write it